Update readme to describe deployment options (#26)
k8s-at-home/container-images: README.md

## Deployment Information

### Deployment on Kubernetes

> Most of these containers are frequently used as drop-in-replacements for our k8s-at-home Helm Charts.

Reviewer: ```suggestion
Most of these containers can be used as drop-in-replacements for the images in our k8s-at-home [Helm charts](https://github.com/k8s-at-home/charts/).
```

While we do not know if (or when) these containers will be the default for the Helm Charts, they work just fine!

> Within docker-compose and other solutions, they should work just fine too. 

Reviewer: ```suggestion
Within docker-compose and other solutions, they should work just fine too but that isn't guaranteed. 
```

We don't want to give the impression of supporting other platforms 😉
